joomla/joomla-cms
# [5.0] Unify the com_ajax functionality

Pull Request for Issue # .

### Summary of Changes

- Add the missing method as `method` part for the plugins
- Add a conditional so the changes would be 100% B/C 

### Testing Instructions

- Install Joomla from the GH PR.
- Check that Sample data still works
- - Check that the Guided tours still works (now using the method URLParam)

### Actual result BEFORE applying this Pull Request

Plugins where only supporting one method `onAjaxPluginName`

### Expected result AFTER applying this Pull Request

Plugins support also a custom `method` name as modules and templates

### Link to documentations
Please select:


@HLeithner @laoneo
